test/puppetlabs/puppetdb/testutils/services.clj

```diff
@@ -110,11 +110,10 @@
 (defmacro with-puppetdb-instance
   "Convenience macro to launch a puppetdb instance"
   [& body]
-  `(with-redefs [sutils/db-metadata (delay {:database "HSQL Database Engine"
-                                            :version [2 2]})]
+  `(with-redefs [sutils/db-metadata (delay (sutils/db-metadata-fn))]
```

Comment:
Sorry, I'm confused by the `delay`. It looks like it was being used as a crappy thunk. Is there any reason to keep it?